front/app/containers/Admin/projects/project/analysis/Insights/index.tsx

```diff
@@ -23,13 +24,25 @@ const Insights = () => {
   const { data: insights, isLoading } = useAnalysisInsights({
     analysisId,
   });
+
+  const largeSummariesEnabled = useFeatureFlag({
+    name: 'large_summaries',
+    onlyCheckAllowed: true,
+  });
+
   const filters = useAnalysisFilterParams();
-  const { data } = useInfiniteAnalysisInputs({
+
+  const { data: allInputs } = useInfiniteAnalysisInputs({
```

Comment:
Is this using the inputs index endpoint? If so, it seems like `allInputs` is only used to get a count. I guess a separate endpoint to get just the count would be more efficient (this endpoint is probably slow on the backend)?

Reply:
Yes, it's true but it's ok because these responses are cached so the back-end endpoint is only technically called once (which is something we need to do in any case). There's limited performance value in creating a separate endpoint for this and it would increase the complexity of the code, if we did.

Reply:
Agreed with Koen to skip [this for now](https://citizenlabco.slack.com/archives/C05EZTFP46N/p1708423964635989). If we need to handle this case in the future, we will add some logic to do so.